.chloggen/add_interval_processor.yaml

```
@@ -0,0 +1,26 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: "new_component"
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: intervalprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Interval processor is now alpha and is included in the contrib and k8s distributions
+
+# One or more tracking issues or pull requests related to the change
+issues:
+  - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/37501
```

```suggestion
issues: 804
```
This is failing the changelog validation. I think you may just want to refer to this PR, and then link out to the contrib issue from the PR description.